model/private_subnet.rb

```diff
@@ -36,7 +36,7 @@ def display_state
   end
 
   include SemaphoreMethods
-  semaphore :destroy, :refresh_keys, :add_new_nic
+  semaphore :destroy, :refresh_keys, :add_new_nic, :repopulate
```

Comment:
Do we use the `repopulate` semaphore at SubnetNexus?

Reply:
Nice catch, leftover from development